video/out/opengl/context_drm_egl.c

```diff
@@ -201,18 +199,15 @@ static int match_config_to_visual(void *user_data, EGLConfig *configs, int num_c
 
 static EGLDisplay egl_get_display(struct gbm_device *gbm_device)
 {
-    const char *ext = eglQueryString(EGL_NO_DISPLAY, EGL_EXTENSIONS);
+    EGLDisplay ret;
 
-    if (ext) {
-        PFNEGLGETPLATFORMDISPLAYEXTPROC get_platform_display = NULL;
-        get_platform_display = (void *) eglGetProcAddress("eglGetPlatformDisplayEXT");
-
-        if (get_platform_display && strstr(ext, "EGL_MESA_platform_gbm"))
-            return get_platform_display(EGL_PLATFORM_GBM_MESA, gbm_device, NULL);
+    ret = mpegl_get_display(EGL_PLATFORM_GBM_MESA, "EGL_MESA_platform_gbm", gbm_device);
```

Comment:
The existing **mpegl_get_display** is a bit bonkers, but I can follow-up on that in due time. I've got some WIP already